### ISSUE TITLE
[Suggestion] add /carpet setGlobalDefault

### ISSUE THREAD

**Issue by supersaiyansubtlety** [gnembon/fabric-carpet#830]
`/carpet setGlobalDefault <carpetSetting>` would act like `/carpet setDefault <carpetSetting>`, except it would save the setting to `config/carpet/carpet_default.conf` instead of the current world's `carpet.conf`

**Comment by Ghoulboy78** [comment]
Curently there is a default_carpet.conf file which is autogenerated and which can be manually changed, but I agree that this would be a good idea

**Comment by supersaiyansubtlety** [comment]
Yeah I'm aware, Im just tired of having to manually copy/paste/merge my rules into the default config.

**Comment by gnembon** [comment]
wonder if that's too much options to choose from. It wouldn't fly with Apple, that's for sure. But with commands, if you don't see it, its not a bother, so should be fine then.

But then - remove default and list defaults should be handled as well.
